common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -476,74 +478,72 @@ object DotcomRenderingUtils {
-      sectionLabel = article.content.sectionLabelName,
-      sectionUrl = article.content.sectionLabelLink,
-      sectionName = article.metadata.section.map(_.value),
-      subMetaSectionLinks = article.content.submetaLinks.sectionLabels.map(SubMetaLink.apply),
-      subMetaKeywordLinks = article.content.submetaLinks.keywords.map(SubMetaLink.apply),
-      shouldHideAds = article.content.shouldHideAdverts,
-      isAdFreeUser = views.support.Commercial.isAdFree(request),
-      webURL = article.metadata.webUrl,
-      linkedData = linkedData,
-      openGraphData = openGraphData,
-      twitterData = twitterData,
-      config = combinedConfig,
-      guardianBaseURL = Configuration.site.host,
-      contentType = jsConfig("contentType").getOrElse(""),
-      hasRelated = article.content.showInRelated,
-      hasStoryPackage = page.related.hasStoryPackage,
-      beaconURL = Configuration.debug.beaconUrl,
-      isCommentable = article.trail.isCommentable,
-      commercialProperties = commercial.editionCommercialProperties,
-      pageType = pageType,
-      starRating = article.content.starRating,
-      trailText = TextCleaner.sanitiseLinks(edition)(article.trail.fields.trailText.getOrElse("")),
-      nav = nav,
-      showBottomSocialButtons = ContentLayout.showBottomSocialButtons(article),
-      pageFooter = pageFooter,
-      publication = article.content.publication,
+        GUDateTimeFormatNew.formatDateTimeForDisplay(article.trail.webPublicationDate, request), // String
+      webPublicationSecondaryDateDisplay = displayedDateTimes.secondaryDateLine, // String
+      editionLongForm = Edition(request).displayName, // String
+      editionId = edition.id, // String
+      pageId = article.metadata.id, // String
+      format = article.metadata.format.getOrElse(ContentFormat.defaultContentFormat), // ContentFormat
+      designType = designTypeAsString(article.metadata.designType), // String
+      tags = allTags, // List[Tag]
+      pillar = findPillar(article.metadata.pillar, article.metadata.designType), // String
+      isImmersive = article.isImmersive, // Boolean
+      sectionLabel = article.content.sectionLabelName, // String
+      sectionUrl = article.content.sectionLabelLink, // String
+      sectionName = article.metadata.section.map(_.value), // Option[String]
+      subMetaSectionLinks = article.content.submetaLinks.sectionLabels.map(SubMetaLink.apply), // List[SubMetaLink]
+      subMetaKeywordLinks = article.content.submetaLinks.keywords.map(SubMetaLink.apply), // List[SubMetaLink]
+      shouldHideAds = article.content.shouldHideAdverts, // Boolean
+      isAdFreeUser = views.support.Commercial.isAdFree(request), // Boolean
+      webURL = article.metadata.webUrl, // String
+      linkedData = linkedData, // List[LinkedData]
+      openGraphData = openGraphData, // Map[String String]
+      twitterData = twitterData, // Map[String, String]
+      config = combinedConfig, // JsObject
```

Comment:
Note that this one is the one we want to enter into and consider typing its fields explicitly in a future PR

Reply:
Understood, yeah, that's a nice follow up :)